### PR TITLE
Update chart version in bulk-scan-orchestrator prod

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Update chart version in bulk-scan-orchestrator prod

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/common/bsp/bulk-scan-orchestrator.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ spec:
         IDAM_API_URL: "https://idam-api.platform.hmcts.net"
         TRANSFORMATION_URL_DIVORCE: ""
         TRANSFORMATION_URL_FINREM: ""
+        UPDATE_URL_PROBATE: ""
```
Comment (lgonczar) [inline k8s/prod/common/bsp/bulk-scan-orchestrator.yaml:27]: Do you think we need this?

Comment (mdayican) [reply k8s/prod/common/bsp/bulk-scan-orchestrator.yaml:27]: if we want to override , if we do not want to use in prod we need.

Comment (nitinprabhu) [reply k8s/prod/common/bsp/bulk-scan-orchestrator.yaml:27]: Don't think so we need this as it is derived from value yaml

Comment (mdayican) [reply k8s/prod/common/bsp/bulk-scan-orchestrator.yaml:27]: we do not want to use this in prod?

Comment (lgonczar) [reply k8s/prod/common/bsp/bulk-scan-orchestrator.yaml]: [discussion was moved to dev channel]

**Pull request merged**